core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeDDLBuilder.scala

```diff
@@ -21,6 +22,66 @@ class SnowflakeDDLBuilder
       case e => throw new IllegalArgumentException(s"Expected a string literal, got $e")
     }
 
+  override def visitDdlCommand(ctx: DdlCommandContext): ir.Catalog = ctx match {
+    case a if a.alterCommand() != null => a.alterCommand().accept(this)
+    case c if c.createCommand() != null => c.createCommand().accept(this)
+    case d if d.dropCommand() != null => d.dropCommand().accept(this)
+    case u if u.undropCommand() != null => u.undropCommand().accept(this)
+  }
+
+  override def visitCreateCommand(ctx: CreateCommandContext): ir.Catalog = {
+    val methods: Seq[() => ParserRuleContext] = Seq(
+      ctx.createAccount _,
```

Comment:
i'm struggling to understand this syntax, other people would as well

Reply:
i changed to this because it is smaller. It is just a list of functions basically and the first one that returns non null is used. I can:

  - Change to just List( ... )
  - Go back to a big switch (I just found that a bit ugly)
  - Change the grammar such that this rule is split into a few sub-collections that are generally related